### ISSUE TITLE
Tabs

### ISSUE THREAD

**Issue by mega94** [socialAPIS/Instagram-API#12]
Thank you for the similar api, recently such clients are gaining traction. 
Can I get tabs, travel, architecture, style, sports? 
And yet this is an emulation of an android application?

**Comment by socialAPIS** [comment]
Yep it is Possible to get the tabs, I will pin this Issue for feature updates
